src/styles/index.scss

```diff
@@ -2,2 +1,75 @@
+body,
+html {
   margin: 0;
+  height: 100%;
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+$calendar-size: 100px;
+$calendar-gap: $calendar-size / 100;
+$calendar-border: $calendar-size / 100;
+$calendar-padding: $calendar-size / 10;
+$calendar-font-size: 0.3 * $calendar-size;
+$calendar-animation-size: 0.2 * $calendar-size;
+$calendar-flex-width: (
+  (7 * $calendar-size) + ($calendar-gap * 6) + ($calendar-padding * 2)
+);
+$week-days: '-mon', '-tue', '-wed', '-thu', '-fri', '-sat', '-sun';
+$counter: 0;
+$base-spacing: $calendar-size + $calendar-gap;
+
+.calendar {
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $calendar-gap;
+  padding: $calendar-padding;
+  width: $calendar-flex-width;
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #eee;
+    width: $calendar-size;
+    height: $calendar-size;
+    border: $calendar-border solid black;
```

Comment:
The value for border width should be defined in pixels if it's less than 1px. Please check if the calculated value for $calendar-border is indeed generating a visible border and if not, consider specifying a minimum pixel value.